text/0031-integ-testing.md

```diff
@@ -0,0 +1,300 @@
+# Integration testing for CDK apps
+
+* **Original Author(s):**: [@nija-at](https://github.com/nija-at)
+* **Tracking Issue**: [#31](https://github.com/aws/aws-cdk-rfcs/issues/31)
+* **API Bar Raiser**: @{BAR_RAISER_USER}
+
+CDK app developers and construct library authors can write integration
+tests that can be run locally or in their CI/CD pipeline.
+
+## Working Backwards - README.md
+
+### contest
```

Comment:
Haaaaaaaaaaaah!

Reply:
Not a fan of the name... I'd just call this `cdk test`/`@aws-cdk/testing`